tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -128,6 +128,61 @@ def test__create_collection__InvalidParameters(self):
                     self.assertIn(error, response.json["detail"])
 
 
+class TestGetCollections(BaseAuthAPITest):
+    def setUp(self):
+        super().setUp()
+        self.test_collection = dict(
+            name="collection", description="description", contact_name="john doe", contact_email="johndoe@email.com"
+        )
+
+    def test__get_collections_no_auth__OK(self):
```

Comment:
get publish collections

Reply:
implicitly it gets public collections but it is named without "public" because the `visibility` is not specified